tests/Fixtures/TestBundle/Entity/UuidIdentifierDummy.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\Resource;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Custom identifier dummy.
+ *
+ * @Resource
+ * @ORM\Entity
+ */
+class UuidIdentifierDummy
+{
+    /**
+     * @var string The custom identifier.
+     *
+     * @ORM\Column(type="guid")
+     * @ORM\Id
+     */
+    private $uuid;
+
+    /**
+     * @var string The dummy name.
+     *
+     * @ORM\Column(length=30)
+     */
+    private $name;
+
+    /**
+     * @return string
+     */
+    public function getUuid()
+    {
+        return $this->uuid;
+    }
+
+    /**
+     * @param string $uuid
+     */
+    public function setUuid($uuid)
+    {
+        $this->uuid = $uuid;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
```

Comment:
btw as we are on PHP7, why not doing `public function getName(): string` and remove the useless phpdoc instead?